src/js/components/sharedComponents/StayInTouch.jsx

```diff
@@ -55,12 +66,16 @@ const StayInTouch = (pageName) => {
                                 Sign up to receive our release notes to keep up with what’s new on USAspending.gov.
                         </div>
                         <div className="stay-in-touch__second-row-link">
-                            <a
-                                href="mailto:join-usaspending@lists.fiscal.treasury.gov?subject=Yes!%20I'd%20like%20to%20receive%20updates."
-                                onClick={trackLinkSignUp}>
-                                    Sign Up
-                                <FontAwesomeIcon icon={faArrowRight} />
-                            </a>
+                            <Button
+                                copy="Sign Up"
+                                buttonTitle="Sign Up"
+                                buttonSize="md"
+                                onClick={handleSignUp}
+                                buttonType="text"
+                                backgroundColor="light"
+                                imageAlignment="right"
+                                textAlignment="left"
+                                image={<FontAwesomeIcon icon={faArrowRight} style={{ height: '16px', width: '16px' }} />} />
```

Comment:
looks much better.  the width should be 14px and the height 16px

Reply:
done